public/js/util.js

```diff
@@ -179,3 +224,282 @@ $.Util.ajaxCall = function(type, url, form) {
 $.Util.toastr = function(type, message) {
     toastr.type(message);
 }
+
+/* setupDashboard()
```

Comment:
I think code that is reused a lot should go in util.js and then page specific should go somewhere else.